modules/contacts/server/controllers/contacts.server.controller.js

```diff
@@ -476,6 +476,7 @@ exports.contactListByUser = function (req, res, next, listUserId) {
           avatarSource: '$user.avatarSource',
           avatarUploaded: '$user.avatarUploaded',
           emailHash: '$user.emailHash',
+          locationLiving: '$user.locationLiving',
```

Comment:
If I understand correctly, `locationLiving` is an independent information from the hosting offer, isn't it? Somehow related to #395 and the confusion of `locationFrom`, `locationLiving` and `Offer.location`. I don't say that it should be fixed here, just to keep that in mind. :)

Reply:
Yes, correct.

There should probably be third location field, `locationCurrent`, which should really only be coordinates (in the database), and which is even more meaningful for `/profile/USER/contacts`
(And then we can make `locationFrom` a bit less prominent because it's not very relevant for the trustroots experience:)